packages/shared/lib/participation/constants.ts

```diff
@@ -50,19 +50,20 @@ export const STAKING_AIRDROP_TOKENS: { [key in StakingAirdrop]: string } = {
  */
 export const PARTICIPATION_POLL_DURATION = 10 * MILLISECONDS_PER_SECOND
 
+export const STAKING_RESULT_URL: string =
+    'https://raw.githubusercontent.com/iotaledger/participation-events/master/results/staking/'
```

Comment:
It is probably safe to use this base URL, although the original ones we got from Alex are ones with some type of hashed ID ([Assembly](https://raw.githubusercontent.com/iotaledger/participation-events/b6d04e17de570aa5b633ee18b1087b2f9bd48601/results/staking/assembly_01.json), [Shimmer](https://raw.githubusercontent.com/iotaledger/participation-events/886597c7372c406ef6a8bc4df165619da0d82af4/results/staking/shimmer.json)), so I'm thinking it'd be safer to use these in case the staking results were ever messed with on the `master` branch of that repo (very unlikely though).

Reply:
It's master so that we can have one base URL and one temp base URL and then just add the file names to the end. Also to tighten the content-security-policy more, as it will have to come from master, and not a random commit.

I don't think we need to worry about the file changing in the Master branch of one of our own repos.